packages/bluesnap-direct-integration/src/BlueSnapDirectEcpFieldset.tsx

```diff
@@ -0,0 +1,65 @@
+import { LanguageService } from '@bigcommerce/checkout-sdk';
+import React, { FunctionComponent, memo } from 'react';
+
+import { CheckboxFormField, Fieldset, Legend } from '@bigcommerce/checkout/ui';
+
+import BlueSnapDirectNumberField from './BlueSnapDirectNumberField';
+import BlueSnapDirectSelectField from './BlueSnapDirectSelectField';
+
+export interface BlueSnapDirectEcpFieldsetProps {
+    language: LanguageService;
+    useFloatingLabel?: boolean;
+    onPermissionChange: (shopperPermission: boolean) => void;
+}
+
+const BlueSnapDirectEcpFieldset: FunctionComponent<BlueSnapDirectEcpFieldsetProps> = ({
+    language,
+    useFloatingLabel,
+    onPermissionChange,
+}) => {
+    const options = {
+        helperLabel: 'Select an account type',
+        items: [
+            { label: 'Consumer checking', value: 'CONSUMER_CHECKING' },
+            { label: 'Consumer savings', value: 'CONSUMER_SAVINGS' },
+            { label: 'Corporate checking', value: 'CORPORATE_CHECKING' },
+            { label: 'Corporate savings', value: 'CORPORATE_SAVINGS' },
+        ],
+    };
+
+    return (
+        <Fieldset
+            legend={<Legend hidden>Electronic Check (ACH/ECP)</Legend>}
+            style={{ paddingBottom: '1rem' }}
```

Comment:
Is there a reason why we are using inline styles?

Reply:
Let me check it and fix it in next PRs